lib/pbench/test/functional/agent/cli/commands/test_config.py

```diff
@@ -4,7 +4,6 @@
 
 def test_pbench_run(monkeypatch, tmpdir, agent_config, pbench_run):  # noqa F811
     """Use pbench_run value from pbench-agent.cfg file"""
-    monkeypatch.delenv("pbench_run")
     env_pbench_run = os.environ.get("pbench_run")
     assert env_pbench_run is None
```

Comment:
This would be a bit more clear as a single line, since the local variable opens the question of whether there's another use of the variable:

```
assert os.environ.get("pbench_run") is None
```

or maybe even

```
assert "pbench_run" not in os.environ
```

At least, something to think about if you make another pass. 😆

Reply:
sure, I will also merge both commits :)